Translation of readme to English

miguelcarreiro opened the pull request:
readme.md - English version
readme.pt.md - Portuguese version

quetzyg commented on readme.md:
```diff
@@ -1,66 +1,66 @@
+- Mailchimp integrated newsletter;
+- Archive page;
+- Global header/footer info, to make less calls to database;
+- Disable jquery global loading, and keep only in home for slickjs (try to optimize as much as possible).
```
`jquery` -> `jQuery`

quetzyg commented on readme.md:
```diff
@@ -1,66 +1,66 @@
-- 
+## TODO
 
+- htaccess cache for imagens/static;
```
`htaccess` -> `.htaccess`

quetzyg commented on readme.md:
```diff
@@ -1,66 +1,66 @@
+- Menu edition in apresentação > menus;
+- Footer columns edition in apresentação > widgets;
+- Duplicate posts to English (and info);
+- In functions.php exists a snippet that removes functions for editors;
```
- There's a code snippet in the `functions.php` file that disables editor functionality;

**(not sure what the original sentence meant, I assume the above)**

quetzyg commented on readme.md:
```diff
@@ -1,66 +1,66 @@
+- Css files have cache buster (no header.php);
+- If, for some reason wpml is disabled, you must pay attention all called instances;
+- The same for acf, but you would have noticed :) ;
+- jquery is only needed for slickjs in home (optimization tip - call script only in home, it's already called in functions);
```
`jquery` -> `jQuery`

miguelcarreiro commented:
> Left some comments

@quetzyg Thanks for your feedback! I've corrected readme with all your suggestions, and will submit a new commit soon.

About your comment referring to line 53, I'm not the original author, so I've tried to translate this readme without changing the meaning of the phrases, but since I agreed with you, this note was also modified.
